
image_width: 672
image_height: 376
camera_matrix:
  rows: 3
  cols: 3
  dt: d
  data:
    [
      3.507833068601e+02,
      0.000000000000e+00,
      3.454384750906e+02,
      0.000000000000e+00,
      3.508783430712e+02,
      1.668714311752e+02,
      0.000000000000e+00,
      0.000000000000e+00,
      1.000000000000e+00,
    ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  dt: d
  data:
    [
      -1.614113747794e-01,
      8.393165174057e-03,
      -2.133761522356e-03,
      -2.049382339958e-04,
      7.005406246477e-03,
    ]